packages/js/ai/src/hooks/useBackgroundRemoval.test.ts

```diff
@@ -154,11 +166,15 @@ describe( 'useBackgroundRemoval hook', () => {
 		} );
 
 		const { result } = renderHook( () => useBackgroundRemoval() );
-		await expect(
-			act( async () => {
+		try {
+			await act( async () => {
 				await result.current.fetchImage( mockRequestParams );
-			} )
-		).rejects.toThrow( 'API Error' );
+			} );
+		} catch ( error: unknown ) {
```

Comment:
This is kinda awkward and verbose, but I was getting warnings and errors when running tests and this was the best solution I could find.
![Screenshot 2023-10-26 at 16 05 47](https://github.com/woocommerce/woocommerce/assets/6723003/24114a1c-2f3b-4565-9d13-2d40e56b70a6)